Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -47,6 +48,8 @@ public struct Configuration: Equatable {
     public let rules: [Rule]
     public var rootPath: String?              // the root path to search for nested configurations
     public var configurationPath: String?     // if successfully loaded from a path
+    public var hash: Int?
```

Comment:
So, to make sure we're in the same page, do you still think we should make `Configuration` conform to `Hashable`?